src/Avalonia.Visuals/Media/FramebufferContext.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using Avalonia.Media.Imaging;
+using Avalonia.Platform;
+using Avalonia.Rendering;
+
+namespace Avalonia.Media
+{
+    public class FramebufferContext : IDisposable
```

Comment:
What is the purpose of this class? I don't see anything that can't be implemented as an extension method for ILockedFramebuffer

Reply:
This is just for the work in progress to find out what should be exposed by ILockedFramebuffer to write such helper methods.